Assets/Scripts/Interactable/ShipFastTravel.cs

```diff
@@ -21,28 +21,27 @@ public class CheckFastTravel : MonoBehaviour
     public GameObject MainShip;
     public GameObject ShipParent;
 
-    public GameObject[] PlayersInScene;
-
     #endregion
 
     #region Private Variables
 
     private int _count = 0;
```

Comment:
Rename this variable to `_playersInShip` so we know what it is counting

Reply:
It is counting players in scene though